common/datavalues/src/types/type_datetime64.rs

```diff
@@ -88,7 +97,19 @@ impl DataType for DateTime64Type {
     }
 
     fn name(&self) -> &str {
-        "DateTime64"
+        match self.precision {
+            3 => DATETIME64_3,
+            6 => DATETIME64_6,
+            9 => DATETIME64_9,
+            _ => unreachable!(),
+        }
+    }
+
+    fn aliases(&self) -> &[&str] {
+        match self.precision {
+            3 => &["DateTime64"],
+            _ => &[],
```

Comment:
how about precision with `6, 9` ?

Reply:
They already have names, this is to represent aliases.

So `DateTime64(3)` has alias `DateTime64`